Sources/SpeziAccount/SpeziAccount.docc/Account Values/Adding new Account Values.md

```diff
@@ -97,6 +97,26 @@ Therefore, you typically do not need to provide a custom view implementation,
 or you might consider adding `CustomLocalizedStringResourceConvertible` protocol
 conformance to your `Value` type.
 
+However, if your `Value` is not `String`-based, you need to implement your own DataDisplayView. Below is a short code example that demonstrates how to implement a DataDisplayView for which the `Value` is `Int`-based for a HeightValue Key.
+```swift
+public struct DataDisplay: DataDisplayView {
+    public typealias Key = HeightKey
+    private let height: Int
+    public init(_ value: Int) {
+        self.height = value
+    }
```

Comment:
Please make sure that this code sample adheres to our swiftlint guidelines. This means moving the initializer below the `body` property to the end of the `DataDisplay` struct. Make sure to leave an empty line between properties and the `typealias` definition and the initializer respectively.